modules/exploits/windows/local/run_as.rb

```diff
@@ -35,7 +35,7 @@ def initialize(info = {})
         [
           [ 'URL', 'https://msdn.microsoft.com/en-us/library/windows/desktop/ms682431' ]
         ],
-      'DisclosureDate' => 'Jan 01 1999' # Not valid but required by msftidy
+      'DisclosureDate' => 'Jan 01 1999' # Same as psexec -- a placeholder date for non-vuln 'exploits'
```

Comment:
Indeed, it would, and I went for that initially... then got suspicious that psexec didn't.

So, sticking with this 1999 business for now.